crm_switzerland/static/sql/execution_signature.sql

```diff
@@ -0,0 +1,44 @@
+﻿--Définir la signature Française
```

Comment:
The comments should be in English. Please change that. We should especially avoid special characters (like ç) in the code.